pkg/config/config.go

```diff
@@ -100,19 +101,53 @@ func (r IPRouteConfig) Ensure(enabled bool) error {
 func (r IPRuleConfig) Ensure(enabled bool) error {
 	var err error
 	if enabled {
-		err = netlink.RuleAdd(&r.Rule)
-		if os.IsExist(err) {
-			err = nil
+		ruleCount, e := r.exist()
```

Comment:
can we have a helper function, say ensureRuleCount(int), then

if enabled {
  ensureRuleCount(1)
} else {
  ensureRuleCount(0)
}